File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int count1 = 0 ; 
        int count2 = 0 ; 

        for ( int ele : nums ) {
            if(ele == 1 ) count1++ ; 
            if(ele == 2 ) count2++ ; 
        }
        Arrays.fill(nums , 0 ) ; 
        for ( int i = nums.length - (count1 + count2) ; i < nums.length ; i++ ) {
            if(count1 >= 1 ) {
                nums[i] = 1 ; 
                count1-- ; 
            }
            else {
                nums[i] = 2 ; 
            }
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        countZero = 0 
        countOne = 0 
        countTwo = 0 

        for ele in nums :
            if ele == 0 :
                countZero = countZero + 1 
            if ele == 1 :
                countOne = countOne + 1 
            if ele == 2 :
                countTwo = countTwo + 1 

        i = 0 

        while countZero != 0 :
            nums[i] = 0 
            i = i+1 
            countZero = countZero - 1 
        while countOne != 0 :
            nums[i] = 1 
            i = i+1 
            countOne = countOne - 1 

        while countTwo != 0 :
            nums[i] = 2 
            i = i+1 
            countTwo = countTwo - 1